Unexpected crash with `doco serv`

```
> doco serv
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
main.main()
	/Users/paganotoni/go/pkg/mod/github.com/paganotoni/doco@v1.0.13/cmd/doco/main.go:58 +0x474

```

--- a/cmd/doco/main.go
+++ b/cmd/doco/main.go
@@ -55,7 +55,7 @@ func main() {
 	case "help":
 		printHelp(os.Stdout)
 	default:
-		fmt.Printf("Unknown command %s\n", args[1])
+		fmt.Printf("Unknown command %s\n", args[0])
 		fmt.Println("--------")
 		printHelp(os.Stdout)
 	}

